addons/sale_stock/models/account_move.py

```diff
@@ -95,6 +95,7 @@ def _filter_outgoing_sml(ml):
                 'quantity': qty,
                 'uom_name': lot_id.product_uom_id.name,
                 'lot_name': lot_id.name,
+                'id': lot_id.id
```

Comment:
@amoyaux Could you live with that change?  Maybe it can just be lot_id

Reply:
Yep rename it lot_id and add a comment that explain it is used in l10n

Reply:
Done!

Reply:
Ping @amoyaux, could we made this change? What do you think about the comment?